minigalaxy/ui/gametile.py

```diff
@@ -47,11 +48,11 @@ def __init__(self, parent, game, api):
 
         # Set folder for download installer
         self.download_dir = os.path.join(CACHE_DIR, "download")
-        self.download_path = os.path.join(self.download_dir, self.game.name)
+        self.download_path = os.path.join(self.download_dir, self.get_real_name())
 
         # Set folder if user wants to keep installer (disabled by default)
         self.keep_dir = os.path.join(Config.get("install_dir"), "installer")
-        self.keep_path = os.path.join(self.keep_dir, "{}.sh".format(self.game.name))
+        self.keep_path = os.path.join(self.keep_dir, self.get_real_name())
```

Comment:
This way of setting this cannot work. Using get_real_name here and on line 51 adds 4 api requests on the main thread for each game you own. For me that means I will have to wait for more than 1000 requests to the API have been made before the application starts responding again. We shouldn't need that many requests at all.